pynetdicom/events.py

```diff
@@ -421,6 +429,11 @@ def __init__(
                 )
             setattr(self, kk, vv)
 
+    def __str__(self) -> str:
+        """String representation of the class."""
+        return 'Event(event={}, message_id={}, timestamp={}, is_cancelled={})'\
```

Comment:
Use f-strings please

Reply:
This turned out to be more complicated than I had understood, so I just removed this.